media/css/teach/smarton.less

```diff
@@ -1422,88 +1426,382 @@ body.not-firefox .footer-cta {
+        .transition(background-position .275s ease-in-out);
+
+        &:nth-child(1n+3) {
+            clear: both;
+            float: none;
+            margin: 0 auto;
+
+            @media only screen and (min-width: @breakDesktop) {
+                float: left;
+                clear: none;
+                margin: 0 @gridGutterWidth (@baseLine * 2);
+            }
+        }
 
         &:before {
             content: '';
         }
+
+        &:after {
+            .font-size(40px);
+            text-align: center;
+            color: #ddd;
+            content: '\25be';
+            display: none;
+            height: 1em;
+            width: 100%;
+            position: absolute;
+            bottom: 85px;
+            left: 0;
+            z-index: 3;
+            visibility: visible;
+            opacity: 1;
+            .animation(pulse 1.5s infinite);
+        }
+
+        &.show:after {
+            display: none;
+        }
+
+        @media only screen and (max-width: @breakTablet) {
+            float: none;
+            margin: 0 auto 20px;
+        }
     }
 
-    h2, h3, a {
-        color: #fff;
+    .topic-title {
+        .font-size(28px);
+        .transition(margin .25s ease);
+        font-style: italic;
         text-shadow: none;
+        margin: 0 0 25px;
+        position: relative;
+        z-index: 3;
+    }
+
+    .topic-intro {
+        .font-size(14px);
+        color: #888;
+        position: relative;
+        z-index: 3;
+        opacity: 1;
+        display: none;
+        .transition(opacity .25s ease);
+    }
+
+    .inner {
+        .border-box;
+        color: #fff;
+        text-decoration: none;
+        display: block;
+        height: 100%;
+        width: 100%;
+        padding: 140px 20px 20px;
+        position: absolute;
+        top: 0;
+        left: 0;
+        z-index: 3;
+        opacity: 1;
+        @transition: opacity .25s ease, padding-top .25s ease;
+        .transition(@transition);
+    }
+
+    .cta {
+        .trailing-arrow;
+        display: inline-block;
+        padding: .25em 20px;
+        border: 2px solid #fff;
+        border-color: rgba(255, 255, 255, .5);
+        border-radius: 5px;
+        .transition(border-color .2s ease);
+
+        &:hover,
+        &:focus,
+        &:active {
+            border-color: #fff;
+            text-decoration: none;
+        }
+    }
+
+    .shape {
+        position: absolute;
+        left: 0;
+        top: 0;
+        right: 0;
+        z-index: 2;
+
+        path {
+            fill: #fff;
+        }
+    }
+
+    &.has-touch .topic {
+        .topic-title {
+            margin: 0;
+        }
+
+        .topic-intro {
+            opacity: 0;
+        }
+
+        .inner {
+            opacity: 1;
+            padding-top: 140px;
+        }
     }
 
     .topic-tracking {
-        background: @theme-tracking;
+        background: url('/media/img/teach/smarton/tracking/tracking-landing.png') center 160px no-repeat,
```

Comment:
Not a blocker but we can probably add the high-res version of `tracking-landing.png`, `security-landing.png` and `surveillance-landing.png`.